src/api/database/bulk-api.rst

```diff
@@ -456,6 +456,132 @@ Sending multiple queries to a database
     The multiple queries are also supported in /db/_local_docs/queries and
     /db/_design_docs/queries (similar to /db/_all_docs/queries).
 
+.. _api/db/bulk_get:
+
+==================
+``/db/_bulk_get``
+==================
+
+.. http:post:: /{db}/_bulk_get
+    :synopsis: Fetches several documents at the given revision
+
+    This method can be called to query several documents in a bulk. It is well
+    fitted when you want a specific revision of the documents, like replicators
+    do for example, or get the revision history.
```

Comment:
I suggest a rewording to sound more natural, and to fix some grammatical mistakes:

> It is well suited for fetching a specific revision of documents, as replicators do for example, or for getting revision history.